### ISSUE TITLE
Create transfers on the test network

### ISSUE THREAD

**Issue by konradkonrad** [raiden-network/raiden#649]
After deployment of the test network #648 we need infrastructure that is able to initiate transfers through the deployed nodes. Other than #652 this issue is about creating some "background noise" of transfers, not a directed scenario for testing special cases.

### Mode of operation

- for each test network node
- every `$INTERVAL`
- randomly chose a target node from the nodes in the token network
- try to send a transfer of `amount == random.randint(1, 20)`

### Tasklist

- [ ] find a scheduler (e.g. `cron`)
- [ ] supply the nodes with the list of known testnet addresses
- [ ] initiate the transfer via raiden RPCAPI

@ulope please update this issue with more concrete steps before starting

**Comment by ulope** [comment]
This is mostly handled by the scenario player with a "random" configuration.

**Comment by LefterisJP** [comment]
I am closing this in favour of #652
